Makefile

```diff
@@ -0,0 +1,34 @@
+
+build: fetch
+	xcodebuild -project ld/zld.xcodeproj -scheme zld -derivedDataPath build -configuration Release build
+
+abseil-cpp-20200225:
+	curl -# -L https://github.com/abseil/abseil-cpp/archive/20200225.tar.gz | tar xz
+	mkdir $@/build
+	cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_STANDARD=17 -S $@ -B $@/build
+	make -C $@/build -j
+	find $@/build/absl -name '*.a' | xargs libtool -static -o $@/build/libabsl.a
+
+cfe-7.0.1.src:
+	curl -# http://releases.llvm.org/7.0.1/cfe-7.0.1.src.tar.xz | tar xJ
+
+clean:
+	rm -rf abseil-cpp-20200225 build cfe-7.0.1.src dyld-635.2 llvm-7.0.1.src pstl tapi-b920569 tbb
+
+dyld-635.2:
```

Comment:
we'll want to make sure that dyld and tapi are on the best versions (and llvm?). currently, i don't think they are. but that can come later

Reply:
The clang and LLVM versions were picked to match up with that Xcode release, but you are right as we are now running against more modern dependencies we should update this to the current Xcode targets instead.